### DISCUSSION TITLE
How do i add Timestamps on Version 1.2.3

### DISCUSSION THREAD

**Discussion by MisterBird69** [Dadangdut33/Speech-Translate#46] [Q&A] [answered]
![image](https://github.com/Dadangdut33/Speech-Translate/assets/145562812/adb7d4b6-58b9-40bf-8169-339c4ea4c37a)

Where and how di i change the Settings to get such Timestamps?
 This guy used version 1.1.0 with the GPU Version.

I'm using Version 1.2.3 with the CPU Version. How ever, i just get files with few rows and huge ccolumns.
I have not found a solution in the Settings "F2"-> transcribe and/or Settings "F2"->textbook.
Pls help

**Comment by MisterBird69** [comment, marked as answer]
I found the Solution. I just need to open the ".srt" file.
Sorry, my bad
Thank you for your program, it works flawlessly even with the tiny model.

**Reply by Dadangdut33** [reply]
i'm glad that it works :D 

I'm also currently working on 1.3 that would hopefully make this stuff more clear with better onboarding more feature and bug fixes

**Reply by Skannet** [reply]
Am on the latest release and my .srt file contains timestamps but the output is all messed up. There are font color lines everywhere in between the dialogues and each dialogue is being repeated again and again.

The output in .txt file is fine just spelling mistakes no repeat of lines.

**Comment by MisterBird69** [comment]
Du hast gebohnen ! du hast meine liebe gebohnen

Ich bin dein hauptleis



________________________________

**Comment by MisterBird69** [comment]
You dream du
Ja nei





________________________________
